Tests/RespondersTests/RouteResponderTest.swift

```diff
@@ -98,7 +98,7 @@ class RouteResponderTest: XCTestCase {
 
       let expected = "GET /someRoute HTTP/1.1"
 
-      XCTAssertEqual(response.body!, expected.toBytes)
+      XCTAssertEqual(response.body!.toBytes, expected.toBytes)
```

Comment:
I do have my HTTPResponse conforming to the Equatable protocol, so I should be able to create an expectedResponse object in the test and compare the two responses as a whole. This might also limit the redundant setup across multiple tests by having one assertion per expected response. I think I'll try that !